site/src/atomics/spacing.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+`<css-property-name>-<value>-<screensize>`
+
+For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md).
+
+The following `<css-property-names>-<values>-<screensizes>` are supported.
+
+Accepted property names
+
+- `margin`, `padding`
+- `margin|padding-inline`
+- `margin|padding-block`
+- `margin|padding-top|right|bottom|left`
+
+Accepted values
+
+- `none, xs, s, m, l, xl, xxl, xxxl`
+
+Accepted screensizes
+
+- `tablet, desktop, widescreen`
+
+## Usages
+
+Adding uniform spacing
+
+Use `margin` with a size value to add spacing around an element.
+
+```html
+<div class="margin-xs border">XS</div>
+<div class="margin-s border">X</div>
+<div class="margin-m border">M</div>
+<div class="margin-l border">L</div>
+<div class="margin-xl border">XL</div>
+<div class="margin-xxl border">XXL</div>
+<div class="margin-xxxl border">XXXL</div>
+```
+
+Adding horizontal and vertical spacing
```

Comment:
Let's make it a heading -> `### Adding horizontal and vertical spacing`

Reply:
Let's not, let's make it a full sentence.